packages/fetch-api/src/api.ts

```diff
@@ -34,9 +34,7 @@ export class GiphyFetch {
      * @hidden
      */
     private getQS = (options: any = {}) => {
-        const { giphy_pbid: pingback_id } =
-            typeof document !== 'undefined' ? cookie.parse(document.cookie) : ({} as any)
-        return qs.stringify({ ...options, api_key: this.apiKey, pingback_id })
+        return qs.stringify({ ...options, api_key: this.apiKey, pingback_id: getPingbackId() })
```

Comment:
Remove the cookie, use the getPingbackId util

Reply:
🔥